### PR TITLE
Add digitalWallets property to GatewayAccount

### PR DESCRIPTION
**Pull request by iamdevice**

### CODE REVIEW

**Comment by iamdevice** [comment]
php please review

**Comment by iamdevice** [comment]
php please review

File: src/Entities/DigitalWallets/ApplePay.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        return $this->getAttribute('isEnabled');
+    }
+
+    /**
+     * @param bool $isEnabled
+     *
+     * @return ApplePay
+     */
+    public function setIsEnabled(bool $isEnabled): self
```
Comment (ashkarpetin) [inline src/Entities/DigitalWallets/ApplePay.php:45]: Are we using type hints?

File: src/Entities/DigitalWallets/DigitalWallets.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    {
+        return $this->getAttribute('googlePay');
+    }
+
+    /**
+     * @param GooglePay $googlePay
+     *
+     * @return DigitalWallets
+     */
+    public function setGooglePay(GooglePay $googlePay): self
```
Comment (ashkarpetin) [inline src/Entities/DigitalWallets/DigitalWallets.php:49]: Same here, we usually use `$value` without typehint.

**Review by ashkarpetin — APPROVED** [review]

**Review by subenson — APPROVED** [review]
lgtm

**Pull request merged**